tools/docker-isolated/Dockerfile

```diff
@@ -23,8 +23,8 @@ RUN ssh-keygen -q -N "" -t dsa -f /etc/ssh/ssh_host_ecdsa_key
 RUN ssh-keygen -q -N "" -t rsa -f /etc/ssh/ssh_host_rsa_key
 RUN sed -i "s/#UsePrivilegeSeparation.*/UsePrivilegeSeparation no/g" /etc/ssh/sshd_config
 RUN sed -i "s/UsePAM.*/UsePAM yes/g" /etc/ssh/sshd_config
-RUN ssh-keygen -A
```

Comment:
This PR sets up a symlink for the isolated key.

The `awx_tools_1` container generates and writes a key at `/awx_devel/awx/main/expect/`, which is mapped to the shared volume for the source code.

The `awx_isolated_1` container sets up a symlink to point `/root/.ssh/authorized_keys` at the synced public keyfile we generate.